Makefile.config.gpu.cudnn

```diff
@@ -26,12 +26,7 @@ CUDA_DIR := /usr/local/cuda
 # CUDA architecture setting: going with all of them (up to CUDA 5.5 compatible).
 # For the latest architecture, you need to install CUDA >= 6.0 and uncomment
 # the *_50 lines below.
-CUDA_ARCH := -gencode arch=compute_30,code=sm_30 \
-             -gencode arch=compute_35,code=sm_35 \
-             -gencode arch=compute_50,code=sm_50 \
-             -gencode arch=compute_52,code=sm_52 \
-	     -gencode arch=compute_60,code=sm_60 \
-	     -gencode arch=compute_61,code=sm_61
+CUDA_ARCH := -gencode arch=compute_61,code=sm_61
```

Comment:
Do we need to update build.sh too ?

Reply:
This is bad commit from my part, will fix it.